server/app/views/applicant/NavigationFragment.html

```diff
@@ -54,7 +58,7 @@
 <section
   th:fragment="officialGovermentWebsiteBanner"
   class="usa-banner"
-  aria-label="Official website of the United States government"
+  th:aria-label="#{label.officialWebsiteOfUsa}"
   data-testId="governmentBanner"
 >
   <div class="usa-accordion">
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +67]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Blocking: Can we remove aria-hidden from this line instead of adding an aria-label for officialWebsiteOfUsa?

I just tested this locally and I think it's a better experience. The user gets to select the real text instead of the whole banner<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/civiform/civiform/8832?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->